### PR TITLE
Raw word info

### LINKED ISSUE #91: Allow word-level information on raw text
Customers request that they can get word-level information on raw text as well as the formatted text.
The proposed change is to add a new field as a companion to [repeated WordInfo words](https://github.com/cobaltspeech/sdk-cubic/blob/master/grpc/cubic.proto#L438) for the raw text, called `raw_words`.

To enable this, the [RecognitionConfigs](https://github.com/cobaltspeech/sdk-cubic/blob/master/grpc/cubic.proto#L173) will not be changed.  Currently, if `enable_word_time_offsets` or `enable_word_confidence` is true, then we will populate the `RecognitionAlternative.words` field.  Moving forward, if either of these are defined _AND_ [enable_raw_transcript](enable_raw_transcript) is true, then we will also populate the new `RecognitionAlternative.raw_words` field.

### PR DESCRIPTION
**Pull request by jhollowayj**
Should close #91.

### CODE REVIEW

File: grpc/cubic.proto [hunk 1]
```diff
@@ -432,11 +432,17 @@ message RecognitionAlternative {
   // likelihood of the output being correct.
   double confidence = 2;
 
-  // A list of word-specific information for each recognized word.  This is
-  // available only if `enable_word_confidence` or `enable_word_time_offsets`
-  // was set to `true` in the `RecognitionConfig`.
+  // A list of word-specific information for each recognized word in the
+  // formatted text. This is available only if `enable_word_confidence` or
+  // `enable_word_time_offsets` was set to `true` in the `RecognitionConfig`.
```
Comment (happyalu) [inline grpc/cubic.proto:437]:
```suggestion
  // A list of word-specific information for each recognized word in the
  // `transcript` field. This is available only if `enable_word_confidence` or
  // `enable_word_time_offsets` was set to `true` in the `RecognitionConfig`.
```

File: grpc/cubic.proto [hunk 2]
```diff
@@ -432,11 +432,17 @@ message RecognitionAlternative {
...
   repeated WordInfo words = 3;
 
+  // A list of word-specific information for each recognized word in the raw
+  // text. This is available only if `enable_word_confidence` or
+  // `enable_word_time_offsets` was set to `true` _and_ `enable_raw_transcript`
+  // is also set to `true` in the `RecognitionConfig`.
```
Comment (happyalu) [inline grpc/cubic.proto:443]:
```suggestion
  // A list of word-specific information for each recognized word in the
  // `raw_transcript` field. This is available only if `enable_word_confidence` or
  // `enable_word_time_offsets` was set to `true` _and_ `enable_raw_transcript`
  // is also set to `true` in the `RecognitionConfig`.
```

File: grpc/go-cubic/cubicpb/gw/cubic.pb.gw.go-E [hunk 3]
```diff
@@ -0,0 +1,299 @@
+// Code generated by protoc-gen-grpc-gateway. DO NOT EDIT.
```
Comment (happyalu) [inline grpc/go-cubic/cubicpb/gw/cubic.pb.gw.go-E:1]: What is this `.go-E` file for?

**Review by happyalu — CHANGES_REQUESTED** [review]
Requested a minor documentation change.

File: grpc/go-cubic/cubicpb/gw/cubic.pb.gw.go-E (hunk 3 above)
Comment (jhollowayj) [reply grpc/go-cubic/cubicpb/gw/cubic.pb.gw.go-E]: I don't know where that came from.  :/  Removed.

**Review by happyalu — APPROVED** [review]
lgtm

**Review by renaissanceGeek — APPROVED** [review]
:+1:

**Pull request merged**